### ISSUE TITLE
Typora cannot search single Latin character

### ISSUE THREAD

**Issue by singularitti** [typora/typora-issues#1524]
When using search/replace text function in Typora, I found that it cannot search a single Latin character in source mode, I have to type at least 2 characters to trigger the search highlight or replacing. Interestingly, I can search other single characters, like "^", "\", etc. For example,

1. No highlight when searching only "W"
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/25192197/40634751-a393e888-62c4-11e8-987b-7efe674a7e99.png">
2. Highlight is triggered when searching "Wh"
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/25192197/40634771-c5620026-62c4-11e8-8a42-957ef7188046.png">
3. Searching for non-Latin can be triggered
<img width="1050" alt="image" src="https://user-images.githubusercontent.com/25192197/40634785-d7cc7638-62c4-11e8-9a9a-f4bf943f42ba.png">
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/25192197/40634798-df4176c0-62c4-11e8-94b5-cf58e64c4d62.png">
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/25192197/40634804-e6e77410-62c4-11e8-8ae3-33a2088e43f6.png">

Is there a specific consideration for this mechanism?

**Comment by abnerlee** [comment]
We do not provide search for single characters since there will be too many matches, which costs much resource for match and rendering, but usually user cannot find the result he/she wants from so many matches.

@singularitti In which case will you do search for single Latin character?

**Comment by singularitti** [comment]
I see your point. I was just curious why this feature was not added.

I use it when I want to replace some symbols in mathematical formulae, there can be a lot of single $f$s representing the same function.

**Comment by fuzzy76** [comment]
I strongly disagree :-/ I just received a large text that had been OCR'ed. And for some reason the OCR had replaced all "tt" with "Z" (capital Z). And I can't search&replace because of this. Perhaps the two-character limit can be bypassed when the search is case sensitive?

**Comment by singularitti** [comment]
@fuzzy76 Maybe you could try other full-featured editors like [VS Code](https://code.visualstudio.com)?
